roles/variables/vars/main.yml

```diff
@@ -158,6 +158,34 @@ kafka_controller_properties:
       listeners: "{{kafka_controller_listeners['controller']['name']}}://:{{kafka_controller_listeners['controller']['port']}}"
       node.id: "{{ node_id|default(groups.kafka_controller.index(inventory_hostname)|int + 9991) if inventory_hostname in groups.kafka_controller else 0}}"
       inter.broker.listener.name: "{{kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['name']}}"
+  migration_mode:
+    enabled: "{{ migration_enabled }}"
+    properties:
+      zookeeper.connection.timeout.ms: 18000
+      zookeeper.metadata.migration.enable: "true"
+      zookeeper.connect: "{{ groups['zookeeper'] | default(['localhost']) | confluent.platform.resolve_hostnames(hostvars) | join(':' + zookeeper_client_port|string + ',') }}:{{zookeeper_client_port}}{{zookeeper_chroot}}"
+  migration_ssl:
+    enabled: "{{ zookeeper_ssl_enabled and migration_enabled|bool}}"
+    properties:
+      zookeeper.ssl.client.enable: 'true'
+      zookeeper.clientCnxnSocket: org.apache.zookeeper.ClientCnxnSocketNetty
+      zookeeper.ssl.truststore.location: "{{kafka_controller_pkcs12_truststore_path}}"
+      zookeeper.ssl.truststore.password: "{{kafka_controller_truststore_storepass}}"
+  migration_mtls:
+    enabled: "{{ zookeeper_client_authentication_type == 'mtls' and migration_enabled|bool }}"
+    properties:
+      zookeeper.ssl.keystore.location: "{{kafka_controller_pkcs12_keystore_path}}"
+      zookeeper.ssl.keystore.password: "{{kafka_controller_keystore_storepass}}"
+  migration_acls:
+    enabled: "{{ zookeeper_client_authentication_type in ['kerberos', 'digest'] and migration_enabled|bool }}"
+    properties:
+      zookeeper.set.acl: 'true'
+  migration_broker_listener:
+    enabled: "{{kafka_controller_sasl_protocol=='plain' and migration_enabled|bool }}"
```

Comment:
What happens for other SASL protocols.

Reply:
we still do not support SASL SCRAM, and for kerberos, the line just above this is used.

Reply:
In case of Migration, Controller needs to communicate with the Broker. So Broker's listener configs are needed in Controller in SASL Plain